api/include/opentelemetry/trace/trace_state.h

```diff
@@ -97,37 +107,163 @@ class TraceState
     }
   };
 
-  // An empty TraceState.
-  TraceState() noexcept : entries_(new Entry[kMaxKeyValuePairs]), num_entries_(0) {}
+  /**
+   * Returns a newly created Tracestate parsed from the header provided.
```

Comment:
nit: change spelling of `Tracestate` in this comment to `TraceState`?

Reply:
Thanks. changed it now :)